examples/account.rs

```diff
@@ -9,5 +9,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     )
     .await?;
     println!("Account: {:?}", account);
+
+    let transactions = accounts::activity(
+        &client,
+        "13vSgJU5rArGv7SryX9h2n4Rz73LM1Achv1J6eFKgjejoKauPr2",
+    )
+    .into_vec()
```

Comment:
do _not_ do this.. that will try to get _all_ activity for an account. Please specify a min_time of like 1 day if you want to jus tinto_vec the results.. Otherwise, still use min_time and use `take` to just get a few